modules/nf-core/bowtie/build/tests/main.nf.test

```diff
@@ -0,0 +1,56 @@
+nextflow_process {
+
+    name "Test Process BOWTIE_BUILD"
+    script "../main.nf"
+    process "BOWTIE_BUILD"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "bowtie"
+    tag "bowtie/build"
+
+    test("sarscov2 - fasta") {
+
+        when {
+            process {
+                """
+                input[0] = [[id: 'sarscov2'],
+                             file(params.test_data['sarscov2']['genome']['genome_fasta'], checkIfExists: true)
+                           ]
```

Comment:
I'm never sure whether to enforce nicer formatting on these.
```suggestion
                input[0] = [
                    [id: 'sarscov2'],
                    file(params.test_data['sarscov2']['genome']['genome_fasta'], checkIfExists: true)
                ]
```

Reply:
I'm never sure what is nicer formatting on them ;)